packages/@react-spectrum/dialog/stories/DialogTrigger.stories.tsx

```diff
@@ -364,7 +364,7 @@ function render({width = 'auto', ...props}) {
         <ActionButton>Trigger</ActionButton>
         {(close) => (
           <Dialog>
-            <Heading>The Heading</Heading>
+            <Heading id="foo">The Heading</Heading>
```

Comment:
Was this added for a reason?

Reply:
just for testing as mentioned in the description